lms/djangoapps/mobile_api/tests/test_course_info_views.py

```diff
@@ -318,30 +318,7 @@ def test_get_requested_user(self, user_role, username, expected_username, mock_g
         else:
             self.assertIsNone(result_user)
 
-    @ddt.data(
-        ({'is_downloadable': True, 'download_url': 'https://test_certificate_url'},
-         {'url': 'https://test_certificate_url'}),
-        ({'is_downloadable': False}, {}),
```

Comment:
It looks like you deleted a test here that perform the negative test (does your code do the right thing if `is_downloadable` is `False`) and I don't see that you replaced it.

Reply:
Added separate test case for this.